### PR TITLE
Add a tool for scikit-learn feature selection module

### PR DESCRIPTION
**Pull request by qiagu**
***Features***:
1. All classes in the feature_selection module were implemented.
2. Multiple options are available for user to choose/type in an estimator in classes SelectFromModel, RFE and RFECV.

***Issues***:
1. The outputs don't support header/column_names yet. Adding the feature doesn't seem to be simple, considering the inputs can either have header or not.

I'll do more tests and add some test cases. 
Please feel free to give it a try and leave your comment or feedback.

### CODE REVIEW

**Comment by bgruening** [comment]
I like this a lot!

**Comment by qiagu** [comment]
😄 Thanks. I'll make more test cases soon.

File: tools/sklearn/feature_selection.xml [hunk 1]
```diff
@@ -24,13 +24,18 @@ import numpy as np
...
+    level = logging.INFO
+)
+logger = logging.getLogger(__name__)
 
 @COLUMNS_FUNCTION@
 
 input_json_path = sys.argv[1]
 params = json.load(open(input_json_path, "r"))
-
+logger.info(params)
```
Comment (bgruening) [inline tools/sklearn/feature_selection.xml]: can we remove these debug output?

**Comment by bgruening** [comment]
Thanks.

**Comment by qiagu** [comment]
@bgruening Thank you for merging.

**Pull request merged**